src/core/server/server.api.md

```diff
@@ -29,17 +29,18 @@ export type APICaller = (endpoint: string, clientParams: Record<string, any>, op
 export type AuthenticationHandler = (request: KibanaRequest, t: AuthToolkit) => AuthResult | Promise<AuthResult>;
 
 // @public
-export type AuthHeaders = Record<string, string>;
+export type AuthHeaders = Record<string, string | string[]>;
 
 // @public
 export interface AuthResultData {

```

Comment:
nit: can we rename this to `AuthResultParams`? We use the `-Params` convention in several places in core.